receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper_others.go

```diff
@@ -27,3 +27,8 @@ func (s *scraper) recordCPUUtilization(now pcommon.Timestamp, cpuUtilization uca
 	s.mb.RecordSystemCPUUtilizationDataPoint(now, cpuUtilization.Idle, cpuUtilization.CPU, metadata.AttributeStateIdle)
 	s.mb.RecordSystemCPUUtilizationDataPoint(now, cpuUtilization.Irq, cpuUtilization.CPU, metadata.AttributeStateInterrupt)
 }
+
+func (s *scraper) getCPUInfo() ([]cpuInfo, error) {
+	var cpuInfos []cpuInfo
+	return cpuInfos, nil
```

Comment:
Would a new partial scrape error lead to a stop of the receiver? If not, we might want to return an error that explains that cpu info is currently not supported on other systems?

Reply:
Hmm, I would avoid returning an error at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/27445/files#diff-b07d518e5053b194b5772f68b9fa418fd93c029d24d4fef8a9f9d5fd2cb333a6R90 on every scrape. This would mean that if the collector is configured to scrape every 5s we would flood the logs with a log that is not actually super critical? But if there is a common way that collector handles these cases we can do it.

Reply:
ok, I think that makes sense :)